velox/functions/prestosql/tests/BinaryFunctionsTest.cpp

```diff
@@ -747,4 +747,65 @@ TEST_F(BinaryFunctionsTest, fromIEEE754Bits32) {
       fromIEEE754Bits32(hexToDec("0000000000000001")),
       "Input floating-point value must be exactly 4 bytes long");
 }
+
+TEST_F(BinaryFunctionsTest, rpad) {
+  const auto rpad = [&](std::optional<StringView> arg,
+                        std::optional<StringView> key,
+                        std::optional<int32_t> size) {
+    return evaluateOnce<std::string>(
+        "rpad(c0, c1, c2)",
+        makeRowVector(
+            {vectorMaker_.flatVectorNullable<StringView>({arg}, VARBINARY()),
```

Comment:
Use makeNullableFlatVector or better yet a version of evaluateOnce that takes types and values.